### PR TITLE
s3 response crash and deps versions

### PR DESCRIPTION
**Pull request by pharosproduction**
1. Relax deps versions
2. Remove crash and handle s3 response correctly

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -41,14 +41,14 @@ defmodule Arc.Mixfile do
...
-      {:httpoison, "~> 0.8"}, # Required for downloading remote files
-      {:ex_aws, "~> 1.0.0", optional: true},
-      {:mock, "~> 0.1.1", only: :test},
+      {:httpoison, "~> 0.11"}, # Required for downloading remote files
+      {:ex_aws, "~> 1.1", optional: true},
+      {:mock, "~> 0.1", only: :test},
       {:ex_doc, "~> 0.14", only: :dev},
 
       # If using Amazon S3:
-      {:poison, "~> 2.0 or ~> 3.0", optional: true},
```
Comment (princemaple) [inline mix.exs]: why? also, have you tried 3.1?

Comment (pharosproduction) [inline mix.exs]: yes, but Phoenix 1.2.1 doesn't like poison 3.1 so I decided to stick to 2.2

Comment (princemaple) [inline mix.exs]: wouldn't "~> 2.0 or ~> 3.0" still work?

Comment (pharosproduction) [inline mix.exs]: I have changed it already back to 2 and 3

**Comment by PavelVesnin** [comment]
Hi guys. I really need this PR for using 0.11.0 gen_stage. Can you merge it and publish to hex @stavro ?

**Comment by PavelVesnin** [comment]
👍

**Pull request merged**